azure-webapp-maven-plugin/src/test/java/com/microsoft/azure/maven/webapp/handlers/WarArtifactHandlerImplTest.java

```diff
@@ -126,12 +124,25 @@ public void assureWarFileExistedWhenFileNotExist() throws MojoExecutionException
 
     @Test(expected = MojoExecutionException.class)
     public void assureWarFileExistedWhenIsNotAFile() throws MojoExecutionException {
-        final WarArtifactHandlerImpl handlerSpy = spy(handler);
         final File fileMock = mock(File.class);
 
         doReturn("test.war").when(fileMock).getName();
         doReturn(false).when(fileMock).exists();
         doReturn(false).when(fileMock).isFile();
         handlerSpy.assureWarFileExisted(fileMock);
     }
+
+    @Test
+    public void assureWarFileExisted() throws MojoExecutionException {
```

Comment:
it seems "existed" is never used, maybe existing or exists

Reply:
tracked here: https://github.com/Microsoft/azure-maven-plugins/issues/168#issuecomment-401998092

This method will be removed sooner.